PIconnect/PI.py

```diff
@@ -223,6 +234,9 @@ def _current_value(self):
         """Return the last recorded value for this PI Point (internal use only)."""
         return self.pi_point.CurrentValue().Value
 
+    def _update_value(self, af_value_obj, update_mode, buffer_mode):
```

Comment:
I would prefer to call the argument just `value`, when we deprecate support for python 2 I want to add type hints to all code more explicitly.

Reply:
np